requirements.txt

```diff
@@ -1,2 +1,2 @@
-django>=2.2
+django>=3.2
```

Comment:
Why do we keep this at 3.2 but not 4.0?

Reply:
it's a public library on pypi, and we should support an LTS django version :) 

also, backend is still on 3.2